### PR TITLE
Feature/op 635 admission note

### PR DESCRIPTION
**Pull request by tsognong**

### CODE REVIEW

File: src/components/accessories/admission/admissionForm/AdmissionForm.tsx [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+  const diagnosisInList = useSelector(
+    (state: IState) => state.diseases.diseasesIpdIn.data
+  );
+
+  const admissionTypes = useSelector(
+    (state: IState) => state.admissionTypes.allAdmissionTypes.data
+  );
+  const wards = useSelector((state: IState) => state.wards.allWards.data);
+
+  const toOptions = (
```
Comment (axelpezzo) [inline src/components/accessories/admission/admissionForm/AdmissionForm.tsx:44]: Please rename this metho with a more specific name, as for eg: renderOptions or something like that.

File: src/components/accessories/admission/admissionTable/AdmissionTable.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      dispatch(getAdmissionsByPatientId(patientCode));
+    }
+  }, [shouldUpdateTable, dispatch, patientCode, getAdmissionsByPatientId]);
+
+  const formatDataToDisplay = (data: AdmissionDTO[]) => {
+    return data.map((item) => {
+      return {
+        admDate: item.admDate
+          ? format(
+              new Date(isNaN(+item.admDate) ? item.admDate : +item.admDate),
```
Comment (axelpezzo) [inline src/components/accessories/admission/admissionTable/AdmissionTable.tsx:55]: I can't understand this. If adminDate is NaN you pass adminDate in any case, right?

File: src/components/accessories/discharge/dischargeForm/DischargeForm.tsx [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+  }, [dispatch, getDischargeTypes]);
+
+  const diagnosisStatus = useSelector(
+    (state: IState) => state.diseases.diseasesIpdOut.status
+  );
+  const typeStatus = useSelector(
+    (state: IState) => state.dischargeTypes.allDischargeTypes.status
+  );
+
+  const toOptions = (data: (AdmissionTypeDTO | DiseaseDTO)[] | undefined) => {
```
Comment (axelpezzo) [inline src/components/accessories/discharge/dischargeForm/DischargeForm.tsx:163]: As before

Comment (tsognong) [reply src/components/accessories/discharge/dischargeForm/DischargeForm.tsx]: OK

File: src/components/accessories/admission/admissionTable/AdmissionTable.tsx (hunk 2 above)
Comment (tsognong) [reply src/components/accessories/admission/admissionTable/AdmissionTable.tsx:55]: I replaced it by new Date(+item.admDate)

Comment (tsognong) [reply src/components/accessories/admission/admissionTable/AdmissionTable.tsx:55]: as we are expecting a milliseconds date.

**Pull request merged**